### ISSUE TITLE
Unknown inputs for plugin "@sentry/netlify-build-plugin": "deployPreviews"

### ISSUE THREAD

**Issue by bengry** [getsentry/sentry-netlify-build-plugin#30]
Hey,
I want to use the plugin, but when configured with `deployPreviews = false` input, I get this error in Netlify:
```
2:38:27 PM: ────────────────────────────────────────────────────────────────────────
2:38:27 PM:   Plugin "@sentry/netlify-build-plugin" invalid input "deployPreviews"  
2:38:27 PM: ────────────────────────────────────────────────────────────────────────
2:38:27 PM: ​
2:38:27 PM:   Error message
2:38:27 PM:   Unknown inputs for plugin "@sentry/netlify-build-plugin": "deployPreviews"
2:38:27 PM:   Plugin inputs should be one of: "sentryOrg", "sentryProject", "sentryAuthToken", "sentryRelease", "sourceMapPath", "sourceMapUrlPrefix", "skipSetCommits", "skipSourceMaps", "releasePrefix"
2:38:27 PM:   Check your plugin configuration to be sure that:
2:38:27 PM:     - the input name is spelled correctly
2:38:27 PM:     - the input is included in the plugin's available configuration options
    - the plugin's input requirements have not changed
​
2:38:27 PM:   Plugin inputs
2:38:27 PM:   deployPreviews: false
2:38:27 PM:   sentryOrg: brandtotal
2:38:27 PM:   sentryProject: brandtotal-app-frontend
2:38:27 PM: ​
2:38:27 PM:   Plugin details
2:38:27 PM:   Package:        @sentry/netlify-build-plugin
2:38:27 PM:   Version:        1.0.4
2:38:27 PM:   Repository:     git+https://github.com/getsentry/sentry-netlify-build-plugin.git
2:38:27 PM:   npm link:       https://www.npmjs.com/package/@sentry/netlify-build-plugin
2:38:27 PM:   Report issues:  https://github.com/getsentry/sentry-netlify-build-plugin/issues
2:38:27 PM: ​
2:38:27 PM:   Error location
2:38:27 PM:   While loading "@sentry/netlify-build-plugin" from netlify.toml
```

Everything seems to be no typos, and I do see that Netlify are pulling v1.0.4 of this plugin, but looking at the actual [npm package dist](https://www.npmjs.com/package/@sentry/netlify-build-plugin?activeTab=explore) (click "manifest.yml" there), the `deployPreviews` input is **NOT** defined.
Also, the `index.js` file on npm doesn't have any reference to `deployPreviews` either, so I'm thinking that what npmjs.org has as 1.0.4 is not _actually_ 1.0.4.

This, in practice, means that https://github.com/getsentry/sentry-netlify-build-plugin/issues/14 is **not** fixed, at least not for [easy] public consumption.

**Comment by bengry** [comment]
I've forked this repo (https://github.com/brandtotal/sentry-netlify-build-plugin) in the meantime and published it to npm on the meantime with just the name change, if anyone else faces the same issue and needs a workaround until this is fixed.

**Comment by bengry** [comment]
Seems to be resolved in v1.0.5. Closing this issue.
